package.json

```diff
@@ -15,11 +15,12 @@
     "build": "tsc --project tsconfig.build.json",
     "build:clean": "rimraf dist && yarn build",
     "build:docs": "typedoc",
-    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn lint:dependencies",
+    "lint": "yarn lint:eslint && yarn lint:prettier --check && yarn lint:dependencies && yarn lint:changelog",
+    "lint:changelog": "auto-changelog validate",
     "lint:dependencies": "depcheck",
     "lint:eslint": "eslint . --cache --ext js,ts",
-    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn lint:dependencies",
-    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:prettier --write && yarn lint:dependencies",
+    "lint:prettier": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
```

Comment:
Not sure if it makes sense, but maybe we should name these scripts agnostic to the tool it's using, e.g., `lint:format`.

Reply:
`lint:lint` isn't _so_ bad. Maybe `lint:code` could work instead though. Of course Prettier also affects code, but maybe it would be clear enough that code linting and formatting are separate but overlapping tasks.

Reply:
how about `lint:src` or `lint:source`?

Reply:
Hmm, I'm not sure. I'm going to change `lint:prettier` back to `lint:misc` (since it doesn't belong here anyway) and think more about this.